tests/framework/microvm.py

```diff
@@ -827,7 +828,7 @@ def ssh_iface(self, iface_idx=0):
         guest_ip = list(self.iface.values())[iface_idx]["iface"].guest_ip
         self.ssh_key = Path(self.ssh_key)
         return net_tools.SSHConnection(
-            netns=self.jailer.netns,
+            netns=self.netns.id,
```

Comment:
Is the change equivalent to the old one? Before we were assigning the `netns` object, now its `id`

Reply:
Yeah, before it was a string, which is the same as `id`